### ISSUE TITLE
I can't Login

### ISSUE THREAD

**Issue by tasnuva1** [contentful/create-contentful-app#213]
I can't log in. I just can't.   Please, fix this thing.....

*The session expired
Please authenticate again*. 

Please, fix this thing.....
I accidentally publish my API key to the public. --I have to change my API key.

**Comment by Jwhiles** [comment]
Hey @tasnuva1 - could you please provide more specific details about what error you are experiencing, and the how to reproduce it?

**Comment by Jwhiles** [comment]
Hi @tasnuva1 - I'm going to close this ticket as there aren't any actionable details. Feel free to re-open this issue with detailed reproduction steps.

**Comment by tasnuva1** [comment]
https://user-images.githubusercontent.com/62549329/104450000-a8942280-55c9-11eb-8eec-944213383bf7.mp4

https://user-images.githubusercontent.com/62549329/104449922-8d291780-55c9-11eb-99c1-b583f2803290.mp4

I'm extremely sorry for my late response. 
For reproducing it: Just see the videos.

https://user-images.githubusercontent.com/62549329/104450734-c0b87180-55ca-11eb-84ac-4f512751d393.mp4

why this is happening?

**Comment by joelramonph** [comment]
I cant log this  is error 
The session expired
Please authenticate again

**Comment by tomwhita** [comment]
I'm also experiencing this issue.  I'm not able to access Contentful on my main computer (Mac Mini) but I am able to access it on my laptop (Macbook Air).  I have tried using incognito mode and multiple browsers.  

I was logging in via Github credentials but I have tried resetting my password and attempting to login via email to no avail.  I have tried deleting cookies from app.contenful.com.

Each time I attempt to login, I'm redirected to `https://app.contentful.com/?already_authenticated=1` and shown an `ERR_CONNECTION_REFUSED` message.
